From fb71e0f4dd9bfe3feaac9f559fc6d0b88bc9af2b Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 6 Dec 2019 14:20:59 +0000
Subject: [PATCH 0903/1000] MANTIS 0055465 CUSS 1.5.1 Biometrics Interface

new version of CUSS.BIOMETRIC.XSD; identCode is not optional


git-svn-id: svn://localhost/SelfServiceCommon/trunk@1507 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/CUSS.BIOMETRIC.XSD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/CUSS.BIOMETRIC.XSD b/SelfServiceCommon/Massai/idl/Cuss/CUSS.BIOMETRIC.XSD
index c663ed22..e3049263 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/CUSS.BIOMETRIC.XSD
+++ b/SelfServiceCommon/Massai/idl/Cuss/CUSS.BIOMETRIC.XSD
@@ -238,7 +238,7 @@
             <xsd:documentation>Data element for command enrollPerson</xsd:documentation>
         </xsd:annotation>
         <xsd:sequence>
-            <xsd:element name="identCode" type="xsd:base64Binary" minOccurs="0">
+            <xsd:element name="identCode" type="xsd:base64Binary">
                 <xsd:annotation>
                     <xsd:documentation>Base64 encoded identification data (e.g. barcode)</xsd:documentation>
                 </xsd:annotation>
-- 
2.41.0.windows.1

